utl/acc.py

```diff
@@ -6,25 +6,28 @@
 
 __dbfile__ = os.path.dirname(os.path.abspath(__file__)) + '/../data/sitedata.db'
 
+def init():
+    db = sqlite3.connect(__dbfile__)
+    db.execute('CREATE TABLE IF NOT EXISTS users (userid INTEGER PRIMARY KEY, username TEXT, password TEXT);')

```

Comment:
is the execute command the same one that you used when you manually created the table?